src/app/api/services/dot-personas/dot-personas.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { Persona } from '../../../shared/models/persona/persona.model';
+import { Observable } from 'rxjs/Observable';
+
+/**
+ * Provide util methods to get Personas.
+ * @export
+ * @class DotPersonasService
+ */
+@Injectable()
+export class DotPersonasService {
+    constructor() {}
+
+    /**
+     * Return Personas.
+     * @returns {Observable<Language[]>}
+     * @memberof DotPersonasService
+     */
+    get(): Observable<Persona[]> {
+        return Observable.of([
+            { id: '1', label: 'Admin' },
+            { id: '2', label: 'Wealthy Prospect' },
+            { id: '3', label: 'Global Investor' },
+            { id: '4', label: 'First Time investor' }
+        ]);
```

Comment:
this have to be request from the BE in the future, so add A TODO here

Reply:
+1